jobs/mattermost/spec

```diff
@@ -540,10 +541,10 @@ properties:
     default: ""
   mattermost.EmailSettings.SendPushNotifications:
     description: Your Mattermost server sends mobile push notifications to the server specified in PushNotificationServer.
-    default: false
+    default: true
```

Comment:
This is an excellent question, and I'm glad you raised it - I find the Mattermost documentation to be a little lacking on this one. Their documentation seems to imply that `false` is the default value here, but the provided `config/default.json` in their source tree has the value set to `true` - and the documentation says that `push-test` is only for Team Edition users; paying customers can use the (ostensibly better) `push.mattermost.com` service instead.

In short - I'm perfectly happy to put this back to `false` because I certainly can't _guarantee_ that this configuration will work in the simplest case, and I also don't want to be known as "The Guy Who Broke Mattermost-Bosh". :-)